lib/fast_client.js

```diff
@@ -705,8 +710,13 @@ FastClient.prototype.requestComplete = function (request)
 		diff = process.hrtime(request.frq_hrtstarted);
 		latency = mod_jsprim.hrtimeMillisec(diff);
 
-		/* Track the requested RPC methoad. */
-		labels = { 'rpcMethod': request.frq_rpcmethod };
+		/*
+		 * Track the requested RPC method and add labels that were requested by
+		 * the client creator.
+		 */
+		labels = mod_jsprim.mergeObjects(this.fc_metric_labels,
+		    false,
```

Comment:
I think the intent of `mergeObjects` would be to pass `undefined` rather than `false`, but anything false-y works.

nit: Should the `rpcMethod` label here *win* over a user-provided one?  I'm fine ignoring that edge case.

Reply:
> I think the intent of mergeObjects would be to pass undefined rather than false, but anything false-y works.

The docs say "For convenience, any of these objects may be falsey, in which case they will be ignored." and I thought `false` was the falsiest thing I could think of. Should we update the jsprim docs to say that `undefined` is the intent?

> nit: Should the rpcMethod label here win over a user-provided one?

Yeah, I could go either way on that. If we change, we'd want the defaults to be an empty object and the rpcMethod to move to the "overrides". Do you think it's worth doing that?

Reply:
> I thought `false` was the falsiest thing I could think of.

Winner.

> Do you think it's worth doing that?

I think either way is fine. My full nerd inclination is to:
- make defaults falsey
- put `rpcMethod` in "overrides"
- perhaps add a note to the docs that `rpcMethod` is reserved